### PR TITLE
feat: activity card of admin dashboard

### PR DESCRIPTION
**Pull request by debjyoti-pandit**
the UI looks like this - 

image

### CODE REVIEW

File: packages/components/dashboard/activity-card/activity-card.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <CardTitle>{t('Activity')}</CardTitle>
+      </CardHeader>
+      <CardBody className="activity-card-body ">
+        <RecentEventsBody
+          events={{
+            data,
+            loaded,
+            loadError,
+          }}
+        />
```
Comment (SanjalKatiyar) [inline packages/components/dashboard/activity-card/activity-card.tsx:27]: why r u not filtering the event ? make it same as ODF Object card.

**Comment by SanjalKatiyar** [comment]
squash the commits

File: packages/components/dashboard/activity-card/activity-card.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { RecentEventsBody } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { EventKind } from '@openshift-console/dynamic-plugin-sdk-internal/lib/api/internal-types';
+import { useTranslation } from 'react-i18next';
+import { Card, CardBody, CardHeader, CardTitle } from '@patternfly/react-core';
+import { eventsResource } from '../../resources';
+import './activity-card.scss';
+
+const ActivityCard: React.FC = () => {
```
[the remark this answers is not in the document]
Comment (debjyoti-pandit) [reply packages/components/dashboard/activity-card/activity-card.tsx:10]: we don't need it as of now

File: packages/utils/noobaa-utils.ts [hunk 3]
```diff
@@ -170,3 +178,20 @@ export const getDataResources = (obj: BucketClassKind): string[] => {
...
+    NooBaaBucketClassModel.kind,
+    NooBaaObjectBucketClaimModel.kind,
+    NooBaaNamespaceStoreModel.kind,
+  ];
+  if (
+    eventKind !== PersistentVolumeClaimModel.kind &&
+    eventKind !== PersistentVolumeModel.kind
+  ) {
+    const eventName: string = event?.involvedObject?.name;
+    return !_.startsWith(eventName, 'noobaa');
```
Comment (SanjalKatiyar) [inline packages/utils/noobaa-utils.ts:194]:
```suggestion
    return _.startsWith(eventName, 'noobaa');
```

File: packages/utils/noobaa-utils.ts [hunk 4]
```diff
@@ -170,3 +178,20 @@ export const getDataResources = (obj: BucketClassKind): string[] => {
...
+  }
+  return !objectStorageResources.includes(eventKind);
```
Comment (SanjalKatiyar) [inline packages/utils/noobaa-utils.ts:196]:
```suggestion
  return objectStorageResources.includes(eventKind);
```

File: packages/components/dashboard/activity-card/activity-card.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: we don't need rgw but all other noobaa related stuff is required (as per whats been discussed so far)...

[the remark this answers is not in the document]
Comment (debjyoti-pandit) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: as per my understanding goes. [on going activities]( https://github.com/debjyoti-pandit/odf-console/blob/5638375fdbe51688e774ae6d93152c2af7201c63/packages/ocs/dashboards/object-service/activity-card/activity-card.tsx#L43) does only related to rgw

File: packages/utils/noobaa-utils.ts (hunk 3 above)
Comment (debjyoti-pandit) [reply packages/utils/noobaa-utils.ts:194]: https://github.com/red-hat-storage/mcg-ms-console/pull/34#discussion_r895799677

File: packages/components/dashboard/activity-card/activity-card.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: https://github.com/debjyoti-pandit/odf-console/blob/5638375fdbe51688e774ae6d93152c2af7201c63/packages/ocs/dashboards/object-service/activity-card/activity-card.tsx#L57
https://github.com/debjyoti-pandit/odf-console/blob/5638375fdbe51688e774ae6d93152c2af7201c63/packages/ocs/dashboards/object-service/activity-card/activity-card.tsx#L62

[the remark this answers is not in the document]
Comment (cloudbehl) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: @SanjalKatiyar I think we can skip the data resiliency for MCG from dashboard as Yuval never showed us in design and I  never saw this working.

[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: I can see ongoing section https://marvelapp.com/prototype/2e221bb6/screen/85673714 here...

[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: still do we want to skip it @cloudbehl  ?

[the remark this answers is not in the document]
Comment (SanjalKatiyar) [reply packages/components/dashboard/activity-card/activity-card.tsx:11]: it is using `NooBaa_rebuild_progress/100` & `NooBaa_rebuild_time`... not sure where ceph is being used...

File: packages/components/resource-pages/CommonDetails.tsx [hunk 5]
```diff
@@ -74,7 +74,7 @@ export const CommonDetails: React.FC<CommonDetailsSectionProps> = ({
         <div className="row">
           <div className="co-m-pane__body">
             <SectionHeading text={t('Conditions')} />
-            <Conditions conditions={(resource as any).status.conditions} />
+            <Conditions conditions={(resource as any)?.status?.conditions} />
```
Comment (SanjalKatiyar) [inline packages/components/resource-pages/CommonDetails.tsx:77]: not sure why this PR is not showing any merge conflicts here: https://github.com/red-hat-storage/mcg-ms-console/pull/38 merged 3 hours ago...

File: packages/components/dashboard/activity-card/activity-card.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (debjyoti-pandit) [reply packages/components/dashboard/activity-card/activity-card.tsx:51]: @SanjalKatiyar added the ongoing activities

**Pull request merged**